### ISSUE TITLE
Set multiline environment variable

### ISSUE THREAD

**Issue by fruedaCode** [nulldriver/cf-cli-resource#62]
Hi,

I'm trying to set a certificate through environment variables like this.
```yml
- put: new-cf
  params:
     org: my_org
     space: my_space
     command: zero-downtime-push
     manifest: source-snapshot/concourse/manifest.yml
     current_app_name: app-name
     path: artifact-snapshot/*.jar
     environment_variables:
       certificate: |
         -----BEGIN CERTIFICATE-----
         .
         .
         .
         -----END CERTIFICATE-----
```

But I'm getting this error whenever I try a zero-downtime-push:

Is working for single line environment variables.
I have intercepted the the concourse task and instead of manifest.yml file I found this:

```shell
Error: yaml: line 8: could not find expected ':'
Usage:
  yq write [yaml_file] [path] [value] [flags]

Aliases:
  write, w

Examples:

yq write things.yaml a.b.c cat
yq write --inplace things.yaml a.b.c cat
yq w -i things.yaml a.b.c cat
yq w --script update_script.yaml things.yaml
yq w -i -s update_script.yaml things.yaml
yq w --doc 2 things.yaml a.b.d[+] foo
yq w -d2 things.yaml a.b.d[+] foo


Flags:
  -d, --doc string      process document index number (0 based, * for all documents) (default "0")
  -h, --help            help for write
  -i, --inplace         update the yaml file inplace
  -s, --script string   yaml script for updating yaml

Global Flags:
  -t, --trim      trim yaml output (default true)
  -v, --verbose   verbose mode
```

**Comment by patrickcrocker** [comment]
Ok, I can reproduce this in my integration tests.  I'm now looking into the best way to fix this!

**Comment by patrickcrocker** [comment]
Fixed in [v2.14.0](https://github.com/nulldriver/cf-cli-resource/releases/tag/v2.14.0).
